modules/ezcontent_paragraphs/modules/ezcontent_referenced_card/ezcontent_referenced_card.module

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ezcontent_referenced_card_preprocess_paragraph__referenced_card(&$variables) {
+  $variables['#attached']['library'][] = 'ezcontent_referenced_card/ezcontent_referenced_card_css';
+}
+
+/**
+ * Implements hook_theme().
+ */
+function ezcontent_referenced_card_theme($existing, $type, $theme, $path) {
+  return [
+    'node_article_card' => [
+      'template' => 'node--article--card',
+      'base hook' => 'node'
+    ]
+  ];
+}
```

Comment:
Please help me understand the requirement of this code.

Reply:
@vaibhavjain-in I have added this code as we need to overwrite the twig for reference card for bartik theme as we are getting the title twice but right now this code is not working so will either fix this code or will remove this code in next PR